lib/computer.rb

```diff
@@ -0,0 +1,7 @@
+class Computer
+
+  def computermove
+  comp = ["Rock", "Paper", "Scissors"].sample
```

Comment:
Really simple way to give a good result of RPS - you've solved it in the simplest way possible which is always great.